contracts/MapleGlobals.sol

```diff
@@ -440,6 +440,14 @@ contract MapleGlobals {
         return validCalcs[calc] && ICalc(calc).calcType() == calcType;
     }
 
+    /**
+        @dev    Returns the lpCooldownPeriod and lpWithdrawWindow.
+        @return lpCooldownPeriod and lpWithdrawWindow.
+    */
+    function getLpParameters() external view returns (uint256, uint256) {
+        return (lpCooldownPeriod, lpWithdrawWindow);
+    }
```

Comment:
Allows pool to fetch lpCooldownPeriod and lpWithdrawWindow in 1 call to globals.

Reply:
This is something I wanna do from the very beginning or even I want to use the struct for some of the data structure so we can easily get in one call

Reply:
Ya, could technically even include the protocol paused state here too, but it would start introducing odd function coupling.